### PR TITLE
Implemented endpoint to create a playlist

### PR DESCRIPTION
**Pull request by devos50**

### CODE REVIEW

**Comment by devos50** [comment]
retest this please

**Comment by devos50** [comment]
@lfdversluis can you review?

File: Tribler/Core/Modules/restapi/channels/channels_playlists_endpoint.py [hunk 1]
```diff
@@ -58,3 +61,51 @@ def render_GET(self, request):
...
+            .. sourcecode:: javascript
+
+                {
+                    "created": True
+                }
+
+            :statuscode 400: if you are missing the name and/or description parameter
+            :statuscode 404: if the specified channel does not exist
+        """
+        channel_info = self.get_channel_from_db(self.cid)
```
Comment (lfdversluis) [inline Tribler/Core/Modules/restapi/channels/channels_playlists_endpoint.py]: I would do the query checks first, then you save a call to the DB if something is off.

**Comment by lfdversluis** [comment]
Comments given

**Comment by devos50** [comment]
retest this please

**Comment by devos50** [comment]
(that was due to failing credit mining tests)

**Comment by devos50** [comment]
retest this please

**Comment by devos50** [comment]
@lfdversluis can you review?

**Comment by devos50** [comment]
@lfdversluis did a review when he was at my place.

**Comment by devos50** [comment]
@whirm can you review?

**Comment by whirm** [comment]
retest this please

**Comment by devos50** [comment]
retest this please

**Comment by devos50** [comment]
```
Traceback (most recent call last):
  File "C:\Python27\lib\threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "C:\workspace\jenkins\workspace\GH_Tribler_PR_tests_win64\tribler\Tribler\Core\Utilities\twisted_thread.py", line 55, in _reactor_runner
    reactor.run(installSignalHandlers=False)
  File "C:\Python27\lib\site-packages\twisted\internet\base.py", line 1194, in run
    self.mainLoop()
  File "C:\Python27\lib\site-packages\twisted\internet\base.py", line 1203, in mainLoop
    self.runUntilCurrent()
--- <exception caught here> ---
  File "C:\Python27\lib\site-packages\twisted\internet\base.py", line 825, in runUntilCurrent
    call.func(*call.args, **call.kw)
  File "C:\workspace\jenkins\workspace\GH_Tribler_PR_tests_win64\tribler\Tribler\Core\APIImplementation\threadpoolmanager.py", line 50, in caller
    fun(*args, **kwargs)
  File "C:\workspace\jenkins\workspace\GH_Tribler_PR_tests_win64\tribler\Tribler\Core\Libtorrent\LibtorrentMgr.py", line 482, in do_dht_check
    lt_session = self.get_session()
  File "C:\workspace\jenkins\workspace\GH_Tribler_PR_tests_win64\tribler\Tribler\Core\Libtorrent\LibtorrentMgr.py", line 186, in get_session
    if hops not in self.ltsessions:
exceptions.TypeError: argument of type 'NoneType' is not iterable
```

**Comment by devos50** [comment]
retest this please

**Comment by devos50** [comment]
```
test_remotedownload (Tribler.Test.test_remote_search.TestMixedRemoteSearch) ... Unhandled Error
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 754, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/jenkins/workspace/GH_Tribler_PR_tests_linux@2/tribler/Tribler/Core/Utilities/twisted_thread.py", line 55, in _reactor_runner
    reactor.run(installSignalHandlers=False)
  File "/usr/lib/python2.7/dist-packages/twisted/internet/base.py", line 1194, in run
    self.mainLoop()
  File "/usr/lib/python2.7/dist-packages/twisted/internet/base.py", line 1203, in mainLoop
    self.runUntilCurrent()
--- <exception caught here> ---
  File "/usr/lib/python2.7/dist-packages/twisted/internet/base.py", line 825, in runUntilCurrent
    call.func(*call.args, **call.kw)
  File "/home/jenkins/workspace/GH_Tribler_PR_tests_linux@2/tribler/Tribler/Core/Libtorrent/__init__.py", line 27, in <lambda>
    lambda_f = lambda a = args, kwa = kwargs: invoke_func(*a, **kwa)
  File "/home/jenkins/workspace/GH_Tribler_PR_tests_linux@2/tribler/Tribler/Core/Libtorrent/__init__.py", line 28, in invoke_func
    download.session.lm.threadpool.add_task(lambda_f, 1)
exceptions.AttributeError: 'NoneType' object has no attribute 'add_task'
```

**Comment by devos50** [comment]
retest this please

**Comment by devos50** [comment]
@whirm tests passed

**Comment by devos50** [comment]
@whirm can you review? :)

**Pull request merged**